views/deposits.slim

```diff
@@ -1,4 +1,9 @@
 h1 Deposits
-form id = "deposits_form"
-    input type="text" placeholder="Amount Deposited"
-    button type="button" submit
\ No newline at end of file
+div id = "form_contents"
+    form class = "deposits_form"
+        input type="text" placeholder="Name" 
+        br    
+        input type="text" placeholder="Amount Deposited"
+        br
+        button [onclick="window.location.href = '/deposits' " type="button" ]
```

Comment:
what were we supposed to do on click 🤔 ? This was our step 0 if you remember well

![aww-board (1)](https://user-images.githubusercontent.com/584211/79358527-fa7c1f00-7f49-11ea-852e-d0fa84e80ded.png)

Reply:
yes, but i should it not be on the button?

Reply:
my problem was how to call the function from here where the button exists